Gemfile

```diff
@@ -20,6 +20,7 @@ gem 'rails-i18n'
 gem 'rails_safe_tasks', '~> 1.0'
 
 gem "activerecord-import"
+gem "db2fog", github: "openfoodfoundation/db2fog", branch: "rails-5"
```

Comment:
Would it be worth applying the last two commits of this fork here?
https://github.com/ceresfairfood/db2fog/commits/master

We don't need all those dependencies.

Reply:
Ah, nice. Yeah, we can do that.